OutOfSchool/OutOfSchool.DataAccess/Models/SubordinationStructure/InstitutionFieldDescription.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace OutOfSchool.Services.Models.SubordinationStructure
+{
+    public class InstitutionFieldDescription : IKeyedEntity<Guid>
+    {
+        public Guid Id { get; set; }
```

Comment:
do we need guid for every entity here? if some of those won't be exposed directly - we can use longs maybe?